src/pages/vendor/VendorEditProductPage/FinishForm/index.tsx

```diff
@@ -0,0 +1,237 @@
+import { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { Box } from '@mui/system';
+
+import Oval from 'src/assets/icons/addProduct/oval.svg';
+import { urls } from 'src/common/constants';
+import StyledButton from 'src/components/shared/StyledButton';
+import {
+  PaddingVariants,
+  StyleVariants,
+} from 'src/components/shared/StyledButton/types';
+import StyledInput from 'src/components/shared/StyledInput';
+import {
+  InputPaddingVariants,
+  InputStyleVariants,
+} from 'src/components/shared/StyledInput/types';
+import useToast from 'src/components/shared/toasts/components/ToastProvider/ToastProviderHooks';
+import {
+  OnboardingHeader4,
+  OnboardingText,
+} from 'src/pages/OnboardingPage/styles';
+import { useUpdateProductMutation } from 'src/redux/addProduct/addProductService';
+import {
+  decreaseAddProductStep,
+  setPrice,
+  resetAddProduct,
+  selectProductId,
+} from 'src/redux/addProduct/addProductSlice';
+import { useAppDispatch, useAppSelector } from 'src/redux/hooks';
+import theme from 'src/theme';
+
+import RejectEditingModal from '../RejectEditingModal';
+
+import OnboardingHeader3 from './styles';
+
+function FinishForm() {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const { showToast } = useToast();
+  const navigate = useNavigate();
+
+  const productId = useAppSelector(selectProductId);
+  const productName = useAppSelector((state) => state.addProduct.name);
+  const productDescription = useAppSelector(
+    (state) => state.addProduct.description
+  );
+  const productSize = useAppSelector((state) => state.addProduct.size);
+  const productBrand = useAppSelector((state) => state.addProduct.brand);
+  const productColors = useAppSelector((state) => state.addProduct.colors);
+  const productMaterial = useAppSelector((state) => state.addProduct.material);
+  const productCategories = useAppSelector(
+    (state) => state.addProduct.categories
+  );
+  const productStyle = useAppSelector((state) => state.addProduct.style);
+  const productType = useAppSelector((state) => state.addProduct.type);
+
+  const productPrice = useAppSelector((state) => state.addProduct.price);
+
```

Comment:
we can use destructuring {price, style, type, etc.} = useAppSelector(state => state.addProduct);

Reply:
fixed